ui/package.json

```diff
@@ -24,9 +24,11 @@
     "url": "https://github.com/apache/cloudstack/issues"
   },
   "scripts": {
+    "prebuild": "./prebuild.sh",
```

Comment:
@shwstppr could relative paths cause any issue? For ex. break in deb/rpm builds?

Reply:
@rhtyd